cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -399,11 +404,12 @@ def test_change_network_security_groups_per_page_items():
             2.Open Azure Network Manager
             3.Select Network Security Groups
             4.Change items per page
-
-    Bugzilla:
-        1524443
     """
-    pass
+    view = navigate_to(provider, "NetworkSecurityGroup")
+    view.toolbar.view_selector.select('List View')
+    for item_count in [10, 20, 50]:
```

Comment:
If there are only 9 security groups, then the tests of 20/50 are fairly useless. Consider covering this BZ with an assertion in the context of a more in-depth test covering the Security Group page behavior.